src/directives/automation/editor/panel/dpEditorPanelWhatsapp.js

```diff
@@ -162,6 +196,7 @@
         scope.headerVariables = scope.selectedComponent.template.variables.filter(({type}) => type === 'header');
         scope.bodyVariables = scope.selectedComponent.template.variables.filter(({type}) => type === 'body');
         iframeRef.src = scope.selectedComponent.template.publicPreviewUrl || "";
+        scope.multimediaType = multimediaConstraint[scope.selectedComponent.template.headerType || 'TEXT'];
```

Comment:
I am not sure about having an unhandled type in the `multimediaConstraints` array:
```
TEXT: {
          acceptedFileTypes: '',
          maxSize: 0, //Mb
        }
```
Wouldn't it be better to set `scope.multimediaType` to null when the file type doesn't match any in the array?

Reply:
We discussed it with @ck-makingsense, and I misunderstood the use of the array. It is used to identify the _template type_ instead of the _file type_.